virtual_ecosystem/models/soil/soil_model.py

```diff
@@ -307,7 +305,6 @@ def _setup(
         """Function to setup up the soil model."""
 
         self.model_constants = model_constants
-        self.core_constants = core_constants
```

Comment:
Is this change because the core constants are now always an attribute of the BaseModel, so by inheritance they are a property of the SoilModel? (or am I misunderstanding this change?)

Reply:
`core_constants` were _always_ a property of the base model and present in the soil model by inheritance 😄 

https://github.com/ImperialCollegeLondon/virtual_ecosystem/blob/8ad15e114a0c7500592f5e30b8b24cbe668e5aaf/virtual_ecosystem/core/base_model.py#L244-L245

Reply:
Ahh whoops 🫠